ui/src/modules/plans/models/operation/type/operation-alias.models.ts

```diff
@@ -18,7 +18,7 @@ import { OperationDetails, OperationPropertiesAlias } from 'spline-api'
 import { SdWidgetExpansionPanel, SdWidgetSchema, SdWidgetSimpleRecord, SplineDataViewSchema } from 'spline-common/data-view'
 import { SgNodeControl } from 'spline-shared/graph'
 
-import { EventOperationProperty } from '../operation-property.models'
+import { ExtraPropertyValuePrimitive, primitivePropsToDvs } from '../operation-property.models'
```

Comment:
@TerranceKhumalo-absa what does it mean `dvs`?

Reply:
@korel-san  It could mean "Data view schema" as the method takes primitive properties and converts them to SdWidgetSchema. Should I rename the method?